test-apps/display-performance-test-app/README.md

```diff
@@ -55,6 +55,8 @@ When you chose to produce images through an 'image' or 'both' test, the program
 
 You can specify which saved view you wish to use for each test with the viewName property.  If external saved views exist for the local file you can specify which external saved view to use with the extViewName property (you can also specify external saved views by name using the viewName property if the name does not clash with a normal saved view).  You can also specify a saved view to use via the viewString property.  The viewString property must include the _viewname and _viewStatePropsString properties (and optionally the _selectedElements and/or _overrideElements properties) which you can copy from the external saved view file.
 
+If you wish to specify multiple saved views in the same model, you can use a asterisk wildcard instead of a specific view name to test multiple views matching the given wildcard. For example, to test all views, set viewName to "*". To test all views with the word 'edge' in them, set viewName to "*edge" or "edge*". Additionally, you can use the savedViewType setting to "external", "internal", or "both" to specify if you only want to test all of the external saved views, local saved views, or both the local and external saved views. If nothing is specified, the savedViewType will default to using both sets of saved views.
```

Comment:
> To test all views with the word 'edge' in them, set viewName to "*edge" or "edge*"

Are you saying `*edge` and `edge*` are equivalent? Wouldn't you use `*edge*` to test all views with the word "edge" in them?

Reply:
So, currently, using that regex function that I've just copied directly from the reviewAndApproval, it's searching for any substring in the view name that matches the given wildcard, so "*edge", "edge*", and "*edge*" would all do the same thing. 

I think if I change it to use this regex function that I found here: https://stackoverflow.com/questions/26246601/wildcard-string-comparison-in-javascript it would act like you are expecting (which is what I had originally intended to do, until last meeting when it was brought up that there was a Bentley made function that did, theoretically, the same thing--though it doesn't seem to be exactly the same, apparently). But I'm all for changing it to act the way you're expecting if you think that's okay--which is more standard wildcard handling, anyways.

//Short code
function matchRuleShort(str, rule) {
  var escapeRegex = (str) => str.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, "\\$1");
  return new RegExp("^" + rule.split("*").map(escapeRegex).join(".*") + "$").test(str);
}

Reply:
Hmm. What would `ed*ge` do? Or `visible*edge*`?

Reply:
Ugh, so the old way is weird. For "ed*ge", it doesn't accept "edge", "1edge", or "visible edge", but "*ed*ge" does accept all of those. However, ""*edge", "edge*", and "*ed*ge*" all also accept all three of the above cases. For "visible*edge", it seems to have trouble with spaces, maybe? It doesn't accept 'visible edge 1', which I would have expected to work.

For the new way, everything seems to work as expected. "ed*ge" finds only things starting with "ed" and ending with "ge", and "edge*" finds everything that starts with "edge" and can have anything after it. The new way, though, IS case sensitive, so it wouldn't find any of the above cases if "*EDGE*" were used.

Reply:
I've gone ahead and updated things to use the new way. I can easily change it back, or add something to the new way to ignore case-sensitivity if you think that'd be superior, though. :)

Reply:
Can you convert the wildcard and the view names to lowercase so it won't be case-sensitive?

Reply:
I think lowercasing is a good idea, and using the new code.  I tried running the old branch and found how/what it matched very confusing.
I also found a couple of new problems though, which I have sent email about.

Reply:
Wildcards are now case-insensitive, so "*edge" will pickup anything ending in "edge", "Edge", "EDGE", etc. :) I think I fixed almost all of your other problems, @DStradley, except for the scaling thing? Need more details on that to reproduce and fix it (unless it's magically fixed somehow with these other changes, lol).